src/components/spreadsheet/utils/config-tables.js

```diff
@@ -734,7 +740,8 @@ export const TABLES_DEFINITIONS = {
                 valueGetter: (params) =>
                     params?.data?.ratioTapChanger?.loadTapChangingCapabilities,
                 cellRenderer: BooleanCellRenderer,
-                editable: isEditable,
+                editable: (params) =>
+                    isEditable(params) && hasTwtRatioTapChanger(params),
```

Comment:
Maybe wrap this condition inside a function as it is done for other fields ?

Reply:
This condition is used only here